src/app/shared/toast/toast.component.pug

```diff
@@ -3,7 +3,7 @@
     .toast-icon
       i.icon([ngClass]="{ 'icon-success' : type == 'success', 'icon-error' : type == 'error', 'icon-info': type == 'info' }")
     div.toast-message
-      div#toast-text([innerHTML]="stringHTML")
+      div#toast-text([innerHTML]="stringHTML", (focus)="pauseTimer()", (blur)="resumeTimer()", tabindex="10")
```

Comment:
Because other interactive elements in the toast have tabindex 10, I have to keep them together. FYI, on asset page we have lots of numbered tabindex, I tried to remove them in the tab order fix but they are very spread out in lots of components, so it requires huge effort if we want to make it all good.